### PR TITLE
Update license to match the PDF.js license

### PR DESCRIPTION
**Pull request by villager-sap**
Update to match the [PDF.js](https://mozilla.github.io/pdf.js/) license - Apache.

### CODE REVIEW

**Comment by villager-sap** [comment]
@donal-tobin-sap @devinea @jlongie: Could you please review?

**Comment by ghost** [comment]
👍

**Comment by devinea** [comment]
:+1:

**Comment by detobin** [comment]
:-1: use correct license https://spdx.org/licenses/

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "Embed PDF.js viewer into your angular application",
   "version": "1.0.0",
   "dependencies": {},
-  "license": "proprietary",
+  "license": "Apache",
```
Comment (detobin) [inline package.json]: Should be `Apache-2.0` https://spdx.org/licenses/

Comment (villager-sap) [inline package.json]: Thanks, wasn't sure where to reference from!

**Comment by villager-sap** [comment]
@detobin: Re-review please? 😺

**Comment by detobin** [comment]
:+1:

**Pull request merged**